recipe/run_test.py

```diff
@@ -34,24 +33,26 @@
 import os
 import subprocess
 from pprint import pprint
-from os.path import isfile
 
-print('sys.executable: %r' % sys.executable)
-print('sys.prefix: %r' % sys.prefix)
-print('sys.version: %r' % sys.version)
-print('PATH: %r' % os.environ['PATH'])
-print('CWD: %r' % os.getcwd())
+pprint('sys.executable: %r' % sys.executable)
+pprint('sys.prefix: %r' % sys.prefix)
+pprint('sys.version: %r' % sys.version)
+pprint('PATH: %r' % os.environ['PATH'])
+pprint('CWD: %r' % os.getcwd())
 
-from distutils.spawn import find_executable
-from distutils.core import setup
-from distutils.extension import Extension
+try:
+    from setuptools import setup, Extension
+except ImportError:
+    from distutils.core import setup, Extension
```

Comment:
I would just use setuptools. There is no reason to rely directly on distutils.

Reply:
Done